liri.js

```diff
@@ -0,0 +1,207 @@
+		console.log("You've chosen: "+inquirerResponse.commands+". Please follow instructions!");
+		var commands = JSON.stringify(inquirerResponse.commands);
+		switch (commands) {
+			case '["my-tweets"]':
+			console.log("twitter");
+			var inquirerTwitter = require("inquirer");
+			inquirerTwitter
+			.prompt([
+					{
+				      type: "input",
+				      message: "Please type a twitter username",
+				      name: "username"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				twitter(inquirerResponse.username);
+			})
+			break;
+
+			case '["spotify-this-song"]':
+			var inquirerSpotify = require("inquirer");
+			inquirerSpotify
+			.prompt([
+					{
+				      type: "input",
+				      message: "Please the name of a song",
+				      name: "song"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				spotify(inquirerResponse.song);
+			})
+			break;
+
+			case '["movie-this"]':
+			var inquirerMovie = require("inquirer");
+			inquirerMovie
+			.prompt([
+					{
+				      type: "input",
+				      message: "Please type a movie name",
+				      name: "movie"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				movie(inquirerResponse.movie);
+			})
+			break;
+
+			case '["do-what-it-says"]':
+			doIt();
+			break;
+
+			default:
+			console.log("Bye!")
+	}
+	});
+};
+
+function twitter(user){
+	var client = new Twitter ({
+		consumer_key : keys.twitterKeys.consumer_key,
+		consumer_secret : keys.twitterKeys.consumer_secret,
+		access_token_key : keys.twitterKeys.access_token_key,
+		access_token_secret : keys.twitterKeys.access_token_secret,
+	});
+	var params = {
+		screen_name: user,
+		count: 20};
+	client.get('statuses/user_timeline', params, function(error, tweets, response) {
+	  if (!error) {
+	  	for (var i = tweets.length - 1; i >= 0; i--) {
+	  		log((i+1)+") "+tweets[i].created_at.substring(0,19)+ " "+ tweets[i].text);
+	  	}
+	  }
+	  else log(error);
+	});
+};
+
+function spotify(song) {
+	log("song: "+ song);
+	var spotify = new Spotify({
+	  id: keys.spotifyKeys.client_id,
+	  secret: keys.spotifyKeys.client_secret
+	});
+	if(song == "") { 
+		song = "The Sign";
+	}
+
+	spotify.search({ type: 'track', query: song }, function(err, data) {
+	  if (err) {
+	    return log('Error occurred: ' + err);
+	  };
+	  for (var i = 0; i < data.tracks.items.length; i++) {
+	  	log("============= "+(i+1)+" ==========================");
+	  	log("                                      ");
+	  	log("Artist: "+data.tracks.items[i].artists[0].name);
+	  	log("Song: "+data.tracks.items[i].name);
+	  	if(data.tracks.items[i].preview_url){
+	  		log("Preview: "+data.tracks.items[i].preview_url);
+	  	};
+	  	log("Album: "+data.tracks.items[i].album.name);
+	  	log("                                      ");
+	  }
+	});
+};
+
+function movie(movieName) {
+	if(movieName == ""){
+		movieName = 'Mr. Nobody.';
+	}
+		var queryUrl = "http://www.omdbapi.com/?t=" + movieName + "&y=&plot=short&apikey=40e9cece";
```

Comment:
this can just be
```javascript
var movieToSearch = movieName.length > 0 ? movieName : "Mr. Nobody";
var queryUrl = "http://www.omdbapi.com/?t=" + movieToSearch + "&y=&plot=short&apikey=40e9cece";